packages/mesh/src/Mesh.js

```diff
@@ -107,6 +107,8 @@ export default class Mesh extends Container
          */
         this.vertexDirty = 0;
 
+        this._transformID = -1;
```

Comment:
btw why -1?

Reply:
Same as in `Sprite`. 

`transform.worldID` is always non-negative.